pandas/tests/resample/test_time_grouper.py

```diff
@@ -143,7 +140,20 @@ def test_aaa_group_order():
                           df[4::5])
 
 
-def test_aggregate_normal():
+@pytest.mark.parametrize('func, assert_func', [
+    ('min', assert_frame_equal),
+    ('max', assert_frame_equal),
+    ('prod', assert_frame_equal),
+    ('var', assert_frame_equal),
```

Comment:
i think you could move all of the func to a fixture in pandas/tests/resample/conftest (might be usable elsewhere)

Reply:
so that the fixture is reusable, std has been added to the downsample methods in test_base.py and sem, median, ohlc, quantile and nunique added to the test_aggregate_normal test, although ohlc fails.